yb-voyager/cmd/sqlfile_parser_test.go

```diff
@@ -0,0 +1,150 @@
+/*
+Copyright (c) YugabyteDB, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func setupFile(objType string, sqlContent string) (*os.File, error) {
+	tmpFile, err := os.CreateTemp("", fmt.Sprintf("%s.sql", strings.ToLower(objType)))
```

Comment:
how does os.CreateTemp works, providing `""` as dir will pickup which directory?

Reply:
It picks some temp kind of folder automatically e.g. `/var/folders/hz/jwtd4ll10515m40x2bb1kx4h0000gn/T/table.sql75800918`